pkg/apis/clusteroperator/types.go

```diff
@@ -348,6 +359,9 @@ type MachineSetConfig struct {
 type MachineSetSpec struct {
 	// MachineSetConfig is the configuration for the MachineSet
 	MachineSetConfig
+
+	// Version references the clusterversion the machine set is running.
+	Version ClusterVersionReference
```

Comment:
According to https://github.com/kubernetes/community/blob/master/contributors/devel/api-conventions.md#object-references, the name of this field should be `ClusterVersionRef`.